gnustep.base fails with undefined symbol: xmemdup

This blocks nixpkgs-unstable from advancing.
https://hydra.nixos.org/build/109464669/nixlog/1
```
checking for gcc... clang
checking whether the C compiler works... no
configure: error: in `/build/gnustep-base-1.26.0':
configure: error: C compiler cannot create executables
See `config.log' for more details
builder for '/nix/store/hm1p3p0jx93vqrx2rbws0h8yra73496x-gnustep-base-1.26.0.drv' failed with exit code 77
```

cc @ashalkhakov  @matthewbauer

--- a/pkgs/desktops/gnustep/base/default.nix
+++ b/pkgs/desktops/gnustep/base/default.nix
@@ -9,6 +9,7 @@
 , libobjc, libgcrypt
 , icu
 , pkgconfig, portaudio
+, libiberty
 }:
 let
   version = "1.26.0";
@@ -30,8 +31,14 @@ gsmakeDerivation {
     libobjc libgcrypt
     icu
     portaudio
+    libiberty
   ];
   patches = [ ./fixup-paths.patch ];
+
+  # Bump to gcc9 has give missing xmemdup symbols. Need libiberty here
+  # to resolve it, unclear why. See #76927 for more info
+  NIX_LDFLAGS = "-liberty";
+
   meta = {
     description = "An implementation of AppKit and Foundation libraries of OPENSTEP and Cocoa";
   };

